### PR TITLE
compute update and viz error fix

### PR DESCRIPTION
**Pull request by vadthyavath**
# Description

- Fix for Deprecated error with standard nc6 compute: Updated the compute with standard nc6s_v3
- visualization error due to recent updates to Pillow package (earlier pillow version was 9.x.x, now it is 10.0.0): using matplotlib to display image now
- successful runs:
- multiclass run: https://github.com/Azure/azureml-examples/actions/runs/5554257986
- multilabel run: https://github.com/Azure/azureml-examples/actions/runs/5554262663
- IS run: https://github.com/Azure/azureml-examples/actions/runs/5554265416

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb [hunk 1]
```diff
@@ -1376,7 +1376,7 @@
     "# visualize explanations of the first image against one of the class\n",
     "img_bytes = base64_to_img(predictions[0][\"visualizations\"])\n",
     "image = Image.open(BytesIO(img_bytes))\n",
-    "display(image)"
+    "plt.imshow(image)"
```
Comment (rjaincc) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb:1379]: We use plt.imshow at other places as well?

File: sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb [hunk 2]
```diff
@@ -448,7 +448,7 @@
     "from azure.ai.ml.entities import AmlCompute\n",
     "from azure.core.exceptions import ResourceNotFoundError\n",
     "\n",
-    "compute_name = \"gpu-cluster-nc6\"\n",
+    "compute_name = \"gpu-cluster-nc6s\"\n",
```
Comment (raduk) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb:451]: nit: maybe gpu-cluster-nc6sv3

File: sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb [hunk 3]
```diff
@@ -537,7 +537,7 @@
     "    compute_config = AmlCompute(\n",
     "        name=compute_name,\n",
     "        type=\"amlcompute\",\n",
-    "        size=\"Standard_NC6\",\n",
+    "        size=\"Standard_NC6s_v3\",\n",
```
Comment (raduk) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb:540]: how about the v2 notebook for OD?

**Review by raduk — CHANGES_REQUESTED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb (hunk 3 above)
Comment (vadthyavath) [reply sdk/python/jobs/automl-standalone-jobs/automl-image-instance-segmentation-task-fridge-items/automl-image-instance-segmentation-task-fridge-items.ipynb:540]: OD has already updated with nc6sv3 https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb

**Review by raduk — APPROVED** [review]

**Review by raduk — APPROVED** [review]

**Review by rjaincc — APPROVED** [review]

**Pull request merged**